Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -0,0 +1,252 @@
+     doesn't simply pickup ``az login`` credentials. It's recommended to
+     setup a dedicated service principal and use it
+   - with the GCP provider ``openshift-install`` will only work with a service
+     account key, which has to be set using ``GOOGLE_APPLICATION_CREDENTIALS``
+     environment variable (e.g. ``GOOGLE_APPLICATION_CREDENTIALS=service-account.json``)
+
+Create an OpenShift OKD Cluster
+===============================
+
+First, set cluster name:
+
+.. code:: bash
+
+   CLUSTER_NAME="cluster-1"
+
+Now, create configuration files:
+
+.. note::
+
+   The sample output below is showing the AWS provider, but
+   it should work the same way with other providers.
+
+.. code:: shell-session
+
+   $ openshift-install create install-config --dir "${CLUSTER_NAME}"
+   ? SSH Public Key ~/.ssh/id_rsa.pub
+   ? Platform aws
+   INFO Credentials loaded from default AWS environment variables 
+   ? Region eu-west-1
+   ? Base Domain openshift-test-1.cilium.rocks
+   ? Cluster Name cluster-1
+   ? Pull Secret [? for help] **********************************
+
+And set ``networkType: Cilium``:
+
+.. code:: bash
+
+   sed -i 's/networkType:\ OVNKubernetes/networkType:\ Cilium/' "${CLUSTER_NAME}/install-config.yaml"
+
+Next, generate OpenShift manifests:
+
+.. code:: bash
+
+   openshift-install create manifests --dir "${CLUSTER_NAME}"
+
+Now, define ``cilium`` namespace:
+
+.. code:: bash
+
+   cat << EOF > "${CLUSTER_NAME}/manifests/cluster-network-03-cilium-namespace.yaml"
+   apiVersion: v1
+   kind: Namespace
+   metadata:
+     name: cilium
+     annotations:
+       # node selector is required to make cilium-operator run on control plane nodes
+       openshift.io/node-selector: ""
+     labels:
+       name: cilium
+       # run level sets priority for Cilium to be deployed prior to other components
+       openshift.io/run-level: "0"
+       # enable cluster logging for Cilium namespace
+       openshift.io/cluster-logging: "true"
+       # enable cluster monitoring for Cilium namespace
+       openshift.io/cluster-monitoring: "true"
+   EOF
+
+.. include:: k8s-install-download-release.rst
+
+Next, render Cilium manifest:
+
+.. parsed-literal::
+
+   helm template |CHART_RELEASE|  \\
+      --namespace cilium \\
+      --set config.ipam=cluster-pool \\
+      --set global.cni.binPath=/var/lib/cni/bin \\
+      --set global.cni.confPath=/var/run/multus/cni/net.d \\
+      --set global.ipam.operator.clusterPoolIPv4PodCIDR=10.128.0.0/14 \\
+      --set global.ipam.operator.clusterPoolIPv4MaskSize=23 \\
+      --set global.nativeRoutingCIDR=10.128.0.0/14 \\
+      --set config.bpfMasquerade=false \\
+      --set global.endpointRoutes.enabled=true \\
+      --output-dir "${OLDPWD}"
+   cd "${OLDPWD}"
+
+.. note::
+
+   If you make any changes to CIDRs in ``"${CLUSTER_NAME}/install-config.yaml"`` you need to make
+   sure that ``clusterPoolIPv4PodCIDR`` and ``clusterPoolIPv4MaskSize`` reflect those changes.
```

Comment:
Let's add a sentence:

> Make sure that the `clusterPoolIPv4PodCIDR` does not conflict with any other addresses in the same VPC.

Reply:
I changed this a little more, it makes sense to show what OpenShift config looks like.